docs/components.md

```diff
@@ -21,28 +21,29 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-<!-- FIXME: wrong in a lot of ways and therefore misleading -->
-
-    User       Virt API      VM TPR    Virt Controller    k8s      VM Pod
-    createVM  --> |                           |            |
-                  o create --> +              |            |
-                  |            | <~ create ~> ?            |
-                  |            |              |            |
-                  |            |              o create --> o ------> +
-                  |            |              |            |         |
-                  |            | <---------------------- get VM Spec o
-                  |            o ----------------------------------> |
-                  |            |              |            |         o defineVM()
-                  |            |              |            |         o watchVM()
-                  |            |              |            |         |
-    deleteVM -->  o delete --> *              |            |         |
-                  |              <~ delete ~> ?            |
-                  |                           |            |         |
-                  |                           o delete --> o ------> *
-                  |                           |            |
-                  :                           :            :
-
-    Legend: ?: Event notification
+    Client        Virt API      VM TPR  Virt Controller    k8s      VM Pod
+    ------------- ------------- ------- ------------------ -------- ------
+
+                  listen <------------- WATCH /vms
+                                           |
+    POST /vms ----> validate               |
+                    create ---> VM ---> observe
+                      |                    v
+                      |                 POST /pods ----> validate
+                      |                    |             create --> Pod
+                      |                    |                         |
+                      |                    |                         v
+                      | <-----------------------------------------> GET /vms
+                      |                    |                        defineVM
+                      |                    |                        watchVM
+                      v                    |                         |
+    DELETE /vms --> validate               v                         |
+                    delete ---> VM ---> observe                      |
+                      |                    v                         |
+                      |                 DELETE /pods --> validate    |
```

Comment:
Here the virt-controller is currently not involved. Altough virt-controller has some cleanup code, to get rid of stale pods, when recreating a VM, only virt-handler does the delete.

So currently, virt-handler will see the VM delete on the apiserver and will shutdown or destroy the domain, which will move the pod to "succeeded" state.